README.md

```
@@ -1,214 +1,837 @@
 
-- Handler may return `[data, err]` to short-circuit success/error, or throw — builder:
-  - catches throws and returns `[null, error]`
-  - validates returned data/error against provided schemas
+### Error Type Guards
+Isn't it amazing? You will never see `Error` or `try/catch` blocks again, and everything is typed top to bottom, well-known and intuitive.
 
-## Typed errors
+But wait, there's more: **Type Guards**!
 
-The typed errors are accessible via the handler arguments (it's always the last argument). Optionally, you can provide `errorsFirst: true` option to the `zagora` call to change that and the typed error helpers will always be the first argument.
+### Error Type Guards
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)